### ISSUE TITLE
Multiple Subtitles streams

### ISSUE THREAD

**Issue by Duckbuster** [dularion/streama#357]
Currently its possible to upload multiple subtitles, but no way to select which one you want.
Would be a nice feature to be able to select multiple subtitle tracks.

**Comment by dularion** [comment]
Ok so this feature requires a bit of work with the subtitles. Currently, the files attached to a movie or episode are split based on their extension as **video files** and **subtitle files**. Then, of all the subtitles, the first is used and immediately passed into the` <video>` tag. In order to have a subtitle-switcher, all subtitles should be loaded into the video and then set as active/inactive based on the selection. example: 

```javascript
<video id="video" controls preload="metadata">
   <source src="video/sintel-short.mp4" type="video/mp4">
   <track label="English" kind="subtitles" srclang="en" src="captions/vtt/sintel-en.vtt" default>
   <track label="Deutsch" kind="subtitles" srclang="de" src="captions/vtt/sintel-de.vtt">
   <track label="Español" kind="subtitles" srclang="es" src="captions/vtt/sintel-es.vtt">
</video>
```

But for this we might need to add the `label` and the `srclang` as extra fields in the` DataFile.groovy`, so that they are editable. 

Finally, the video-player frontend needs to be adjusted to allow subtitle switching, currently there is just a toggle-button for on/off. 
![image](https://cloud.githubusercontent.com/assets/936076/25852828/feb5208e-34ca-11e7-9035-f8e24c60f579.png)
 Instead, the button needs to open a little popup (similar to the episode-selector)
![image](https://cloud.githubusercontent.com/assets/936076/25852842/0898d640-34cb-11e7-8b6f-ee6a21449e17.png)


The reason why we should add all subtitle tracks to the video is because mobile browsers use the native subtitle-picker in fullscreen mode.

**Comment by DerHerrHerr** [comment]
That's a cool feature,

There is a pull request

**Comment by dularion** [comment]
fixed in v1.1
